lib/checks/lists/only-dlitems.js

```diff
@@ -1,19 +1,41 @@
-var bad = [],
```

Comment:
What is this set of changes doing?

Reply:
@dylanb - these are just changes removed from old pr - https://github.com/dequelabs/axe-core/pull/518